libgap.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@prefix@/include
+
+Name: @PACKAGE_NAME@
+Description: C/C++ library powering GAP, a computer algebra system
+Version: @PACKAGE_VERSION@
+Libs: -L${libdir} -lgap -lgmpxx -lgmp
```

Comment:
```suggestion
Libs: -L${libdir} -lgap -lgmp
```
GAP does not use gmpxx.

I also don't quite understand why `-lgmp` is needed -- isn't this automatically pulled in by `libgap.so`? I don't know much about pkg-config, so this just me being ignorant, and so I am assuming there is a reason -- but it'd be good to know what it is :-)

Reply:
OK I read a bit of the pkg-config docs and looked at some example .pc files, and based on that it seems to me that perhaps it should be either
```suggestion
Libs: -L${libdir} -lgap
```
or maybe
```suggestion
Requires: gmp
Libs: -L${libdir} -lgap
```
??

Although the latter assumes that the name "gmp" in `gmp.pc` is "standard" somehow... which I'd hope it is, but I am not sure if there is something like a global registry for these things?

Reply:
no, there is no global registry for pkg-config package names.